.github/workflows/pr_push.yml

```diff
@@ -205,3 +205,5 @@ jobs:
   MultiNuma:
     needs: [Build]
     uses: ./.github/workflows/multi_numa.yml
+  Nightly:
+    uses: ./.github/workflows/nightly.yml
```

Comment:
this should be removed before merge, right?

Reply:
Yes